base/fhs-validator.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env sh
```

Comment:
wonder if we should make a simple go program like `extensions-validator` and just use everywhere

Reply:
I was think we could also use in talos and extensions

Reply:
For extensions we use extensions-validator. Maybe create some kind of profiles for it? And in Talos we either copy pkgs or install to specific places from dockerfile (i.e. we, not a third party makefile does the install)